internal/driver/glfw/window.go

```diff
@@ -17,9 +17,6 @@ import (
 )
 
 const (
-	scrollAccelerateRate   = float64(5)
-	scrollAccelerateCutoff = float64(5)
-	scrollSpeed            = float32(10)
 	doubleClickDelay       = 300 // ms (maximum interval between clicks for double click detection)
 	dragMoveThreshold      = 2   // how far can we move before it is a drag
 	windowIconSize         = 256
```

Comment:
This is the reason that `goimports` is failing. With the long names removed from the block, it wants the equal signs to be brought more to the left.

Reply:
Yeah, makes sense. Usually VSCode does `gofmt` on save but I think it timed out since it hadn't finished loading the full repo state when I saved the file. Fixed now